packages/fxa-settings/src/pages/ResetPassword/CompleteResetPassword/index.tsx

```diff
@@ -214,11 +217,13 @@ const CompleteResetPassword = ({
         GleanMetrics.resetPassword.createNewSubmit();
 
         const accountResetData = await account.completeResetPassword(
+          keyStretchExperiment.queryParamModel.isV2(),
           token,
           code,
           emailToUse,
           newPassword,
           location.state?.accountResetToken
+          // TODO, we need kB????
```

Comment:
??? Sounds important?   But I think this is the page for resetting without know the current password and without recovery key?

Reply:
Sorry, this was a comment to myself I left during planning. You are corrrect there's a different page for the recovery key.